arch/x86/core/intel64/locore.S

```diff
@@ -724,6 +725,7 @@ irq:
 	cmpl $CONFIG_ISR_DEPTH, ___cpu_t_nested_OFFSET(%rsi)
 	jz 1f
 	sti
+	nop
```

Comment:
Also safe, though a little more subtle.  The nested field in the CPU struct is inc/decremented only with interrupts locked, is properly nested, and is (obviously) CPU-specific.  So we can't see this change without a logic bug elsewhere like a mis-nested interrupt, or something that migrated us to a different CPU somehow.

This too is on interrupt entry, so my vote would be to remove.